common/arg.cpp

```diff
@@ -770,7 +815,7 @@ static std::string common_docker_get_token(const std::string & repo) {
 }
 
 static std::string common_docker_resolve_model(const std::string & docker) {
-    // Parse ai/smollm2:135M-Q4_K_M
+    // Parse ai/smollm2:135M-Q4_0
```

Comment:
The Q4_K_M version of the model seems temperamental, trying to recommend Q4_0 instead.